index.js

```diff
@@ -1,20 +1,28 @@
 'use strict';
 
+const path = require('path');
+const fs = require('fs');
 const EventEmitter = require('events').EventEmitter;
 const requestHandler = require('./lib/request-handler');
 const fetchTemplate = require('./lib/fetch-template');
 const parseTemplate = require('./lib/parse-template');
 const requestFragment = require('./lib/request-fragment');
-const path = require('path');
-const fs = require('fs');
 const PIPE_DEFINITION = fs.readFileSync(path.resolve(__dirname, 'src/pipe.min.js'));
-const AMD_LOADER_URL = 'https://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.22/require.min.js';
 
 module.exports = class Tailor extends EventEmitter {
 
     constructor (options) {
         super();
-        const amdLoaderUrl = options.amdLoaderUrl || AMD_LOADER_URL;
+        const pipeChunk = (amdLoaderUrl, pipeInstanceName) => {
+            let definition;
+            if (amdLoaderUrl.startsWith('http')) {
```

Comment:
This is not a good way. It's possible to have `/` or `//` urls when not serving from CDN